### DISCUSSION TITLE
blog/adding-categories-tags-in-posts/

### DISCUSSION THREAD

**Discussion by giscus** [sujaykundu777/devlopr-jekyll#213] [General]
# blog/adding-categories-tags-in-posts/

Learn how to add categories in posts- Adding Multiple Categories in Posts

https://devlopr-jekyll-git-sk-april2023-sujaykundu777.vercel.app/blog/adding-categories-tags-in-posts/

**Comment by sujaykundu777** [comment]
test

**Comment by liminalnotes** [comment]
nice

**Comment by mystiqmonk** [comment]
Comment from local
